src/ds5/advanced_mode/advanced_mode.cpp

```diff
@@ -239,9 +240,10 @@ namespace librealsense
             []() { STAFactor af; af.amplitude = 0.f; return af; }();
     }
 
-    bool ds5_advanced_mode_base::supports_option(const uvc_sensor& sensor, rs2_option opt) const
+    bool ds5_advanced_mode_base::supports_option(const synthetic_sensor& sensor, rs2_option opt) const
     {
-        return sensor.supports_option(opt);
+        uvc_sensor* raw_uvc_sensor = dynamic_cast<uvc_sensor*>(sensor.get_raw_sensor().get());
```

Comment:
why does you need to get the raw sensor? the synthetic_sensor support the options API

Reply:
nice, thanks